src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1558,13 +1603,8 @@ query_fd_forward_dest(int fd_src)
 static int
 allocate_a_fd_from_kernel(void)
 {
-	struct timespec times_loc;
-	char            file_name[128];
-
-	clock_gettime(CLOCK_REALTIME, &times_loc);
-	snprintf(file_name, sizeof(file_name) - 1, "dummy_%ld_%ld", times_loc.tv_sec,
-		 times_loc.tv_nsec);
-	return memfd_create(file_name, 0);
+	/* Just use open() to allocate a fd from kernel. Not going to read the file. */
+	return open("/proc/self/maps", O_RDONLY);
```

Comment:
This looks like regression so perhaps a compile check would be better here, although looking at it this is only called in dup2 so perhaps this is good enough.